src/apicast-cli/cli/busted.lua

```diff
@@ -0,0 +1,16 @@
+local actions = require "apicast-cli.cmd"
+local colors = require "ansicolors"
+
+local function call(_, parser)
+  local busted_run = parser:command("b busted", "Runs busted tests.")
+    :action(actions.busted.run)
+
+  busted_run:option("-p", "only run test files matching the Lua pattern")
```

Comment:
@mikz @hallelujah 

don't know how to get this option within `src/apicast-cli/cmd/busted.lua` 

i'm doing `bin/apicast-cli b -p somepatter` but `args` are blank.